backend/package.json

```diff
@@ -53,6 +53,7 @@
     "libphonenumber-js": "1.10.6",
     "lodash": "4.17.21",
     "module-alias": "2.2.2",
+    "moment": "^2.29.4",
```

Comment:
by convention, think we're freezing the dependencies (realise I didn't do that for `yamljs` below oops)

Reply:
Will freeze both

Reply:
thanks boss